pymc_bart/pgbart.py

```diff
@@ -160,6 +165,9 @@ def __init__(
             num_observations=self.num_observations,
             shape=self.shape,
         )
+
+        self.linear_fit = fast_linear_fit()
```

Comment:
No need to define this here (and then pass it to the different functions) just call it  inside `draw_leaf_value`

Reply:
removed it https://github.com/pymc-devs/pymc-bart/pull/79/commits/613d0ecc6b65a69dcdf935a562f8ef5d68e385c7